SD_DSC.adoc

```diff
@@ -1916,7 +1916,7 @@ There are no AGD evaluation activities for this component.
 
 *_Root of Trust for Storage_*
 
-Testing for this component is performed through evaluation of FCS_CKM.1, FCS_STG_EXT.1 and FPT_PHP.3.
+Testing for this component is performed through evaluation of FCS_STG_EXT.1 and FPT_PHP.3.
```

Comment:
I think the reason FCS_CKM.1 is here is because this is about how the RoT is created, and part of that could be key generation of some sort (it likely is). This isn't about protecting the SDOs in general, but only those that are used to make the RoT.

Reply:
I don't really understand how this would work in practice, but I'll revert the change.